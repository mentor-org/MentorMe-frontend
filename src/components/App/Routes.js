import React from 'react';
import { BrowserRouter, Route, Switch } from 'react-router-dom';
import Welcome from '../pages/Welcome';


const Routes = () => (
    <BrowserRouter>
        <Switch>
            <Route exact path="/" component={Welcome} />
        </Switch>
    </BrowserRouter>
);

export default Routes;
