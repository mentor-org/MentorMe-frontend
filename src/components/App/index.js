import React, { Fragment } from 'react';
import ReduxToastr from 'react-redux-toastr';
import 'react-redux-toastr/src/styles/index.scss';
import Routes from './Routes';

/**
 * Represents the App Component
 * @returns {component} App
 */
const App = () => 
    <Fragment>
        <Routes />
        <ReduxToastr
            timeOut={3000}
            newestOnTop={false}
            preventDuplicates
            transitionIn="bounceIn"
            transitionOut="fadeOut"
        />
    </Fragment>
;

export default App;
