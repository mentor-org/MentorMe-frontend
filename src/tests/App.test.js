import React from 'react';
import App from '../components/App';

describe('App', () => {
  it('should render the App component correctly', () => {
      const shallowWrapper = shallow(<App />);

      expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
