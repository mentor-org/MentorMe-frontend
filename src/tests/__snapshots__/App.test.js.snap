// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App component correctly 1`] = `
<Fragment>
  <Routes />
  <Connect(ReduxToastr)
    newestOnTop={false}
    preventDuplicates={true}
    timeOut={3000}
    transitionIn="bounceIn"
    transitionOut="fadeOut"
  />
</Fragment>
`;
